cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -53,7 +53,7 @@ func loadConfig(data []byte) (*config.KubeSchedulerConfiguration, error) {
 		// more details.
 		cfgObj.TypeMeta.APIVersion = gvk.GroupVersion().String()
 		if cfgObj.TypeMeta.APIVersion == configv1beta2.SchemeGroupVersion.String() {
-			klog.Warning("KubeSchedulerConfiguration v1beta2 is deprecated in v1.25, will be removed in v1.26")
+			klog.V(0).InfoS("KubeSchedulerConfiguration v1beta2 is deprecated in v1.25, will be removed in v1.26")
```

Comment:
per https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#change-log-functions, klog.Warning should just be klog.InfoS, I think

Reply:
ah fixing

Reply:
i saw this pattern of V(0) else where in the code base ...

Reply:
Must have slipped through code review. It doesn't make much difference, `klog.V(0).InfoS` and `klog.InfoS` are semantically equivalent, the former just has a bit more overhead.